main.go

```diff
@@ -32,7 +34,17 @@ func init() {
 }
 
 func MakeNewRelicPlugin(logger restql.Logger) (restql.LifecyclePlugin, error) {
-	app, err := newrelic.NewApplication(newrelic.ConfigFromEnvironment())
+	app, err := newrelic.NewApplication(
+		newrelic.ConfigFromEnvironment(),
+		func(c *newrelic.Config) {
+			maxSamples := os.Getenv("NEW_RELIC_TRANSACTION_EVENTS_MAX_SAMPLES_STORED")
+			if maxSamples != "" {
+				if m, err := strconv.Atoi(maxSamples); nil != err {
+					c.TransactionEvents.MaxSamplesStored = m
+				}
+			}
```

Comment:
- Can we build something like this? It's easier to add new environment variables.

https://github.com/newrelic/go-agent/blob/198c033a21ef66200032d444e1be06b354175516/v3/newrelic/config_options.go#L123

- Can we make error handling more explicit? Example:

https://github.com/newrelic/go-agent/blob/198c033a21ef66200032d444e1be06b354175516/v3/newrelic/config_options.go#L100

- Maybe `err == nil` instead of `err != nil`

Reply:
Updated with a copy-paste solution with the private function inside the main package.